### PR TITLE
RHDEVDOCS-4375 Update "PAC command reference" topic to reflect CLI breaking changes from RNs

### PR DESCRIPTION
**Pull request by rolfedh**
- Aligned team: Dev Tools
- For branches: 4.11+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-4375
- Direct link to doc preview: 
- SME review: @sm43
- QE review: @ppitonak 
- Peer review: @jc-berger 
- All reviews complete. Please merge now.

### CODE REVIEW

**Comment by sm43** [comment]
LGTM

**Comment by rolfedh** [comment]
/assign @concaf 
Would you QE review and approve this PR?

File: modules/op-pipelines-as-code-command-reference.adoc [hunk 1]
```diff
@@ -65,11 +65,11 @@ If you do not have an {product-title} cluster, it asks you for the public URL th
...
 |===
 
-| Command | Description 
+| Command | Description
 
-| `tkn pac repo create` | Creates a new {pac} repository and a namespace based on the pipeline run template. 
+| `tkn pac create repository` | Creates a new {pac} repository and a namespace based on the pipeline run template.
 
-| `tkn pac repo list` | Lists all the {pac} repositories and displays the last status of the associated runs.
+| `tkn pac list` | Lists all the {pac} repositories and displays the last status of the associated runs.
```
Comment (rolfedh) [inline modules/op-pipelines-as-code-command-reference.adoc:72]: These are the only two lines that need review. 
All the other changes on this page are caused by automated whitespace removal. (The previous contributor didn't have this feature enabled when they created the content.)

Comment (rolfedh) [reply modules/op-pipelines-as-code-command-reference.adoc:72]: /unhold

**Review by concaf — APPROVED** [review]
looks good

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**